Labor camp updates

Xhuis opened the pull request:
### Do not merge.

First off, some screenshots:

_The new, revamped Labor Camp._
![space station 13 2015-10-21 165458](https://cloud.githubusercontent.com/assets/5659269/10649926/abcaad0e-7814-11e5-9a7f-d93e40167445.png)

_An updated cache of goods as well as a new one with a wumborian fugu._
![space station 13 2015-10-21 165513](https://cloud.githubusercontent.com/assets/5659269/10649936/b8ce1040-7814-11e5-9abb-cc49f82b6efa.png)

Now, onto the finer details.
- The Labor Camp Quarters has been added; this area serves as a living space for prisoners and a place for them to gather equipment and rest between mining trips.
- Safety pickaxes have been replaced with capped pickaxes. These pickaxes mine slower and have only 5 brute damage (punching is generally more effective). You can activate them for a 5% chance of removing the caps after 8 seconds, but only if you don't have much bruising.
- Mining scanners have been added to the equipment racks.
- Safety shovels, being deprecated, have been removed.
- Ore is much more plentiful near the labor camp, and laborers will find much more valuable ore while mining.
- The new standard workers' loadout: laborer's overalls, work boots, and labor backpacks. Labor backpacks (sprites by WalterJe) can only hold three items and slow down the wearer.

(Sprite: ![bspkrs](https://cloud.githubusercontent.com/assets/5659269/10643172/6d1df0c2-77ef-11e5-8775-b1ee2c228ee9.png) )
- There is a hidden note under one of the beds in the quarters telling the reader to mine south (the location of loot caches and the ore itself).
- There is a new cache of equipment south of the camp itself. It contains a live wumborian fugu and a unique space suit and helmet. The suit and helmet both cause the wearer to take 30% more brute damage and have obvious names and descriptions. The suit slows down the wearer, and the helmet inflicts severe nearsightedness. The wumborian fugu itself is very hard to defeat with nothing but a capped pickaxe. In addition, the room is surrounded by walls, requiring equipment or gibtonite to destroy.
- The small equipment room with a ushanka and soviet uniform now also has a Syndicate toolbox.
- An escape pod has been added to the Labor Camp. Its floors are brig floors (preventing objective completion) but it still allows prisoners with a way to get out of the Labor Camp if nobody picks them up when the shuttle arrives.

Commit:
Labor camp updates

bgobandit commented:
can there be a way for this ore to ever make it to science

Xhuis commented:
@bgobandit, only if Security comes to pick it up, which is the case by default even before these edits.

bgobandit commented:
no I mean, like, how science gets a "New ores available" message when shit shows up at the cargo machine

Xhuis commented:
No, that only appears with ore redemption machines. The labor camp doesn't have them.

bgobandit commented:
yes that is what I am saying, can a notification be added

Bawhoppen commented:
I doubt anyone will ever use the laborer backpack if theres ANY slowdown.

Xhuis commented:
Say that to the people who use dufflebags.

Commit:
Scatters some emergency O2 closets around the place

Xhuis commented:
In the latest commit:
* Emergency O2 closets have been scattered around the labor camp
* Fixes an undocumented issue on the main Mining outpost where a single tile had the "Space" area

Bawhoppen commented:
>Say that to the people who use dufflebags.


THATS BECAUSE DUFFLEBAGS HOLD A BUNCH OF SHIT, AND EVEN THEN THEY'RE RARELY USED

Xhuis commented:
What I'm trying to say is that people will accept a slowdown for a benefit. On the normal station, backpacks are extremely common and you only get a bit more space for the slowdown. On the labor camp, people may accept a slowdown for three extra storage slots that are surprisingly useful in a place where every item is valuable.

Bawhoppen commented:
3 slots is nothing for that slowdown.

Zilenan91 commented:
Baw, you're assuming people will be fighting on the labor camp. I can see this happening if sec actually used it more, but if you don't intend on fighting, slowdown is manageable and okay. A piece of armor that does it very well is the mining hardsuit. It's spaceworthy and gives you godlike melee protection, but slows you down in atmos. You can stop the slowdown issue by using a jetpack in space, but otherwise, yeah, you're slow, but you're also almost invulnerable to one of the most common sources of damage in the game.

Anonus commented:
Does this fix all the unconnected pipes on the mining zlevel?

Xhuis commented:
@Anonus I'm not aware of any unconnected pipes.

Anonus commented:
There are, unless someone removed them, unconnected pipes under a number of windows near the mining shuttle dock and unconnected pipes under the racks that have pickaxes and shovels in the labor camp.
Are those fixed with this?

Xhuis commented:
I haven't heard of them. Thanks for letting me know - I'll look into fixing them.

AlexCrimson commented:
Remove the spacesuit and toolbox. Sec will never use the labor camp if the prisoners can just escape.

Xhuis commented:
Security who meta to that degree would likely execute the prisoner anyway on the suspicion of them having an uplink implant or hidden objects regardless. In the coming update, the space suit will be behind metal walls anyway, requiring gibtonite or acquisition of the tools beforehand.

Commit:
WIP escape pod, balance change with space suit

Xhuis commented:
In the latest commit:
* Removed lots of disconnected pipes on the mining z-level.
* Added an escape pod to the Labor Camp (this is not finished).
* Surrounded the space suit cache with walls, effectively making equipment or gibtonite a prerequisite for breaching the cache.

Bawhoppen commented:
I really think that there being a suit at all will discourage it's use even more than it already is.

PKPenguin321 commented:
how 2 escape:
0. hope and pray that the emergency o2 closets give you a full set of internals (and optionally, grab a bruise pack). if they do then you're good to proceed
1. meta your way down to the syndie toolbox
2. weld through walls to space suit
3. use 15 brute syndie toolbox to kill wumbo
4. (optional: use your bruise pack to heal up after the fight)
5. profit

how 2 prevent escape (as sec):
1. remove any internals masks from the o2 lockers
2. profit

:+1: for making escape from the gulag possible, while also making it easily preventable.

Anonus commented:
The gulag is barely even used, wasn't used more after it was made impossible to spacewalk to the syndie suit, and this won't make it used any more.
It's basically a prisoner dumping ground, like permabrig but even more permanent.

Xhuis commented:
The main reason it doesn't get any use is because there's no incentive to go there. It's not any _fun_, and there's a lot of mapping glitches there in general. That's why I made this PR - to give the labor camp some love and hopefully encourage its use.

Bawhoppen commented:
The mineral rarity is actually a big change for the gulag, but I still think it won't see much use just due to how security operates. 
Add some abandoned crates.

Zilenan91 commented:
Hey Xhuis can you also make there be a sort of gulag compound between the two buildings? If there's even one space breach it'll depressurize the whole thing and kill everyone, even reaching into the main gulag sec place due to the plastic flaps. You may have to move the ore stuff inside to fix this, to be honest.

Xhuis commented:
@Bawhoppen I believe abandoned crate spawns are determined like ores, or am I wrong?
@Zilenan91 I can easily add some pressurized bridges, yeah.

Bawhoppen commented:
@Xhuis I believe there are crate spawners where theres a chance to spawn a crate, but I may be wrong.

Xhuis commented:
I'll look into it.

bgobandit commented:
@Xhuis I keep saying, if you want people to go there then add something that notifies science when ores are in, otherwise it is literally useless labor

Dorsisdwarf commented:
Make a system which takes ores off the shuttle automatically

Xhuis commented:
Might not be some progress on this for a while, as I'm working on vampires.

AlexanderUlanH commented:
:-1: for a syndie toolbox on the asteroid with everything you'd need to hack your way to freedom without so much as risking burn damage
But an escape method requiring smart and lucky application of gibtonite sounds perfectly reasonable.

Dorsisdwarf commented on code/modules/mining/mine_items.dm:
```diff
@@ -74,6 +75,46 @@
+
+/obj/item/weapon/pickaxe/capped/attack_self(mob/user) //Uncapping!
+	if(!ishuman(user))
+		user << "<span class='warning'>You don't have hands to manipulate [src]!</span>"
```
Monkies. Also, the only other things that could be picking up a pickaxe and using it somehow are xenos, no?

Dorsisdwarf commented on code/modules/mining/mine_items.dm:
```diff
@@ -74,6 +75,46 @@
 
+/obj/item/weapon/pickaxe/capped //Used at the labor camp
+	name = "capped pickaxe"
+	desc = "A pickaxe capped with rubber softeners, considerably dampening its use a weapon. The caps themselves are fitted extremely tightlu."
```
typo

Dorsisdwarf commented:
Why are the quarters so huge for something which will barely be used?

Razharas commented:
@Xhuis 
Tell when its ready

Xhuis commented:
Yep. On a brief hiatus for now but I'll let you know when I ~~FIGURE OUT HOW PODS WORK JESUS FUCK~~ work on this.

Xhuis commented:
In the interest of closing the number of active PRs I have, this is being closed until I decide to continue it, if ever.
